release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3077,6 +3077,8 @@ To avoid this issue, you can enable confidential VMs on an existing cluster by u
 
 * In {product-title} {product-version}, assigning an IPv6 egress IP to a network interface that is not the primary network interface is unsupported. This is a known issue and will be fixed in a future version of {product-title}. (link:https://issues.redhat.com/browse/OCPBUGS-17637[*OCPBUGS-17637*])
 
+* {run-once-operator} (RODOO) can not be installed on Hypershift Operator clusters. This is a known issue and will be fixed in a future version of {run-once-operator}. (link:https://issues.redhat.com/browse/OCPBUGS-17533[*OCPBUGS-17533*])
```

Comment:
should can and not be together like `cannot` ?  Also should we call `Hypershift Operator Clusters` or `Hypershift Clusters` ? Note as such looks good to me.

Reply:
I'll check in docs.openshift to see what we usually call Hypershift.